packages/core/src/renderer/components/dock/logs/list.scss

```diff
@@ -7,10 +7,6 @@
   --overlay-bg: #8cc474b8;
   --overlay-active-bg: orange;
 
-  // fix for `this.logsElement.scrollTop = this.logsElement.scrollHeight`
-  // `overflow: overlay` don't allow scroll to the last line
-  overflow: auto;
-
```

Comment:
Is this no longer a problem then?

Reply:
I don't see any problems for now. It sticks to the bottom line and has proper behavior scrolling to the very top.


https://user-images.githubusercontent.com/9607060/219589366-30576b28-6c1d-4696-8d96-df534a67aedc.mov